ci.mk

```diff
@@ -111,28 +111,30 @@ endef
 #    make tidy
 # @pre the make variable `clang-arch` must be defined if using the tidy rule
 #    with a valid target fot the clang compiler
-# @pre the make variable `CPPFLAGS` must be defined with all pre-processor
-# options, specially the include directory -paths (e.g -I/my/include/dir/inc)
-# @param a single space-separated list of C files (header or source)
+# @param1 a single space-separated list of C files (header or source)
+# @param2 a list of pre-processor options, specially the include directory 
+# -paths (e.g -I/my/include/dir/inc)
 # @example $(call ci, tidy, file1.c file2.c file3.h)
```

Comment:
You should update the example and remove the `(e.g. ...)` from the previous line